.github/workflows/build.yml

```diff
@@ -10,6 +10,17 @@ on:
     branches: [ master ]
 
 jobs:
+  verify-python:
+    name: Python code verification (src)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          pip install black
+      - name: Verify code (python w/black)
+        run: |
+          black --check -t py36 src/
```

Comment:
if it can be configured through a configuration file, then it should be done so that folks running the tool locally prior to checking-in can use the same configuration parameters

Reply:
I added `makefile` with 3 commands. No additional configuration is needed, just `make format-check`.